src/test/kotlin/pt/up/fe/ni/website/backend/controller/ProjectControllerTest.kt

```diff
@@ -58,9 +58,24 @@ internal class ProjectControllerTest @Autowired constructor(
         )
     )
 
+    final val testOldAccount = Account(
```

Comment:
any reason to call it "old"? Seems a bit odd, I think something like testAccount2 would be easier to understand